packages/web/src/pages/artist-dashboard-page/ArtistDashboardPage.tsx

```diff
@@ -86,7 +103,13 @@ export const messages = {
   publicTracksTabTitle: 'PUBLIC TRACKS',
   unlistedTracksTabTitle: 'HIDDEN TRACKS',
   filterInputPlacehoder: 'Filter Tracks',
-  thisYear: 'This Year'
+  thisYear: 'This Year',
+  usdc: 'usdc',
+  earn: 'Earn USDC by selling your music',
+  learnMore: 'Learn More',
+  withdraw: 'Withdraw Funds',
+  salesSummary: 'Sales Summary',
+  withdrawHistory: 'Withdraw History'
```

Comment:
I think sooner rather than later we should all agree on withdrawal vs. withdraw. I believe it's supposed to be the former.

Reply:
dang you right